ONSdigital/eq-author-app
# Ear 1113 refactor comments

### What is the context of this PR?

This PR refactors the comments panel.

I have:

* Created two new components: Comment, and CommentEditor.
* Moved the root CommentPanel into App from Components.

### How to review

Make sure everything is still working as it did before.

- Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
  - ensure it can be opened in Author;
  - then, ensure it can be viewed in Runner by pressing the **view survey** button
- Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
